code/datums/chatmessage.dm

```diff
@@ -210,7 +210,22 @@
 
 	// Approximate text height
 	var/complete_text = "<span class='center [extra_classes.Join(" ")]' style='color: [tgt_color]'>[text]</span>"
-	var/mheight = WXH_TO_HEIGHT(first_hearer.MeasureText(complete_text, null, CHAT_MESSAGE_WIDTH))
+
+	var/mheight
+	WXH_TO_HEIGHT(first_hearer.MeasureText(complete_text, null, CHAT_MESSAGE_WIDTH), mheight)
+
+
+	if(!TICK_CHECK)
+		return finish_image_generation(mheight, target, complete_text, lifespan, extra_classes)
```

Comment:
Explain?

Reply:
if it's not overtiming it continues without queuing it on the subsystem, idk bro I didn't write the code